blocks/article-body-block/chains/article-body/_children/quote.jsx

```diff
@@ -10,7 +11,8 @@ export default ({ element, className }) => {
   contentElements.forEach((contentItem) => {
     if (contentItem.type === 'text'
       && Object.prototype.hasOwnProperty.call(contentItem, 'content')) {
-      quoteItems.push(<p
+      quoteItems.push(<SecondaryFont
```

Comment:
Is this supposed to be Secondary Font? The ticket only mentions updating items to be primary font